src/database.ts

```diff
@@ -828,7 +828,10 @@ class Database extends common.GrpcServiceObject {
         ? (optionsOrCallback as TimestampBounds)
         : {};
 
-    const traceConfig = {opts: this._observabilityOptions};
+    const traceConfig = {
```

Comment:
I really believe that this `traceConfig` creation should move to constructor where we can add all the base configuration and extend it where ever required. Just in this case we had to make code changes at multiple places , it would have been very simple if we would have this in constructor. WDYT ?

Reply:
Sure, I've moved it to the constructor. I would have preferred to do that in another PR but we are already here so let's get it out. Thank you and done!

Reply:
I wired that up for src/database.ts but doing the same for all of them is quite a hassle and a needle in the haystack that I don't think we should spend time on right now. Kindly please take a look at the latest code.